app.vue

```diff
@@ -0,0 +1,27 @@
+<script setup>
+const { data } = await useAsyncData('data', () => queryContent().findOne());
```

Comment:
[Normal]
Usually using descriptive variable names helps to improve code readability.
In this case instead of `data`, you could call variable similar to `webinarSchedule`.
That would help the reader to quickly understand what does the variable contain.